### PR TITLE
Add VS Code samples

### PR DESCRIPTION
**Pull request by jdneo**
Hi JavaFX team, 👋

This PR is to add more JavaFX samples working with Visual Studio Code.

Below sections are added:
- IDE/VSCode/Non-modular
   - Java
   - Maven
   - Gradle

- IDE/VSCode/Modular
  - Maven
  - Gradle

> Currently the VS Code Java has an issue to work with modular project without the build system. Once the issue is fixed, we will update the samples.

Signed-off-by: sheche <sheche@microsoft.com>

### CODE REVIEW

**Comment by nickzhums** [comment]
@jperedadnr could you help to review this one? (assuming you are right person) thanks

**Comment by jperedadnr** [comment]
Thanks for the contribution, we'll have a look shortly.

**Comment by jdneo** [comment]
Hello @jperedadnr @abhinayagarwal, any chance to take a look at this PR?

File: IDE/VSCode/Non-Modular/Gradle/hellofx/src/main/java/org/openjfx/FXMLController.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+
+public class FXMLController {
+
+    @FXML
+    private Label label;
+
+    public void initialize() {
+        label.setText("Hello, JavaFX");
```
Comment (abhinayagarwal) [inline IDE/VSCode/Non-Modular/Gradle/hellofx/src/main/java/org/openjfx/FXMLController.java:12]: Unlike the modular version, this is missing the version information. Is there a reason for this?

File: IDE/VSCode/Non-Modular/Gradle/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+For the first time only:
+
+- Make sure you have installed following extensions in your Visual Studio Code:
+  - [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack)
+  - [Gradle for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-gradle)
+
+- Clone the sample, open the folder `hellofx` in Visual Studio Code.
+
+To run the JavaFX application, you can open the `Gadle Projects` explorer, expand `hellofx` > `Tasks` > `application` and run the Gradle task: `run`
```
Comment (abhinayagarwal) [inline IDE/VSCode/Non-Modular/Gradle/README.md:19]: s/Gadle/Gradle

File: IDE/VSCode/Non-Modular/Gradle/hellofx/src/main/java/org/openjfx/FXMLController.java (hunk 1 above)
Comment (jdneo) [reply IDE/VSCode/Non-Modular/Gradle/hellofx/src/main/java/org/openjfx/FXMLController.java:12]: Ah, I copied it from https://github.com/openjfx/samples/blob/master/IDE/Eclipse/Non-Modular/Gradle/hellofx/src/main/java/org/openjfx/FXMLController.java#L12.

I will update to display the version information.

**Comment by jdneo** [comment]
@abhinayagarwal Thanks for reviewing, I've address the comment. Please take a look when you have time, thanks! 😄

**Review by jperedadnr — COMMENTED** [review]
Looks good overall. Tested all samples on Mac, and left a few comments.

**Comment by jdneo** [comment]
Thank you @jperedadnr, I've updated the PR, please take a look when you have time.

**Review by jperedadnr — APPROVED** [review]
Looks good!

**Comment by jdneo** [comment]
Could we merge this PR now?

**Comment by jperedadnr** [comment]
@jdneo Done!
Are you planing to do a follow up PR for https://github.com/openjfx/openjfx-docs ?

**Comment by jdneo** [comment]
Thank you @jperedadnr!

Yes I'll write the document part recently and send a PR for that.

**Pull request merged**